SignInCheckIn/SignInCheckIn/Unity.config

```diff
@@ -53,6 +64,15 @@
       </constructor>
     </register>
 
+    <!-- Printing REST client -->
+    <register name="printingRestClient" type="RestSharp.IRestClient" mapTo="RestSharp.RestClient">
```

Comment:
You can inject the authenticator like [this example](https://github.com/crdschurch/crds-angular/blob/development/Gateway/crds-angular/Unity.config#L156) from crds-angular